src/core/annotation.js

```diff
@@ -1477,6 +1504,7 @@ class MarkupAnnotation extends Annotation {
     return new this.prototype.constructor({
       dict: annotationDict,
       xref,
+      isOffscreenCanvasSupported: params && params.isOffscreenCanvasSupported,
```

Comment:
Could we perhaps just change https://github.com/mozilla/pdf.js/blob/f7449563ef8722df30c2988fe571236ab9eb89f9/src/core/annotation.js#L316
to
```javascript
InkAnnotation.createNewPrintAnnotation(xref, annotation, {
  isOffscreenCanvasSupported,
});
```
such that we can assume that `params` is always defined and thus do the following here?
```suggestion
      isOffscreenCanvasSupported: params.isOffscreenCanvasSupported,
```